cli/command/swarm/init_test.go

```diff
@@ -1,15 +1,18 @@
 package swarm
 
 import (
-	"fmt"
 	"io/ioutil"
+	"net"
 	"testing"
 
+	"fmt"
+
 	"github.com/docker/cli/internal/test"
```

Comment:
nit:  placement of "fmt" seems to violate go idioms -- i.e. it should be up before "io/ioutil" so it is in alphabetical order.

Reply:
done